### PR TITLE
Fix/107 Refactored acat module code

### PR DESCRIPTION
**Pull request by gonzcm**
## Description
Refactored ACAT module

## Related Issue(s)
- #107 

## Checklist
- [x] I have read and followed the project's contributing guidelines.
- [x] I have tested the changes locally.
- [x] I have made corresponding changes to the documentation if necessary.
- [x] My code follows the project's code style guidelines.

## Reviewers
- @alblopnov

### CODE REVIEW

**Review by alejandromd — APPROVED** [review]
The refactoring is correct. Well done!

File: src/modules/acat/intervention/service.py [hunk 1]
```diff
@@ -1,29 +1,30 @@
...
 
 
-async def get_interventions_service(db: DataBaseDep) -> list[Intervention]:
-    return await Intervention.get_multi(db, query=None)
+async def get_interventions_service(db: DataBaseDep) -> list[model.Intervention]:
+    return await model.Intervention.get_multi(db, query=None)
 
 
-async def create_intervention_service(db: DataBaseDep, intervention_data: InterventionCreate, patient_data: Patient) -> InsertOneResultMongo:
+async def create_intervention_service(db: DataBaseDep, intervention_data: model.InterventionCreate, patient_data: patient_model.Patient) -> InsertOneResultMongo:
```
Comment (alblopnov) [inline src/modules/acat/intervention/service.py:22]: This line is too long for pep8, please divide it into 2 lines

**Review by alblopnov — CHANGES_REQUESTED** [review]
You have a long line in the intervention service

File: src/modules/acat/intervention/service.py (hunk 1 above)
Comment (gonzcm) [reply src/modules/acat/intervention/service.py]: Solved!

**Review by alblopnov — APPROVED** [review]
Thank you for fixing it! Everything look good now!

**Pull request merged**